### PR TITLE
RaR: Eavesdrop

### PR DESCRIPTION
**Pull request by msbeck**
Implemented the ICE subroutine addition like "Sub Boost" and the trace out of an operation from "Manhunt".

### CODE REVIEW

File: src/clj/game/cards/operations.clj [hunk 1]
```diff
@@ -383,6 +383,25 @@
                                    :effect (req (if tagged
                                                   (damage state side eid :meat 1 {:card card})
                                                   (tag-runner state :runner eid 1)))}}}}}
+   
+   "Eavesdrop"
+    (let [new-sub {:label "[Eavesdrop]: Trace 3 - Give the Runner 1 tag"}]
+      {:implementation "On encounter effect is manual"
+       :sub-effect {:label "Give the Runner 1 tag"
+                   :delayed-completion true
```
Comment (nealterrell) [inline src/clj/game/cards/operations.clj]: One more space 😄

File: src/clj/game/cards/operations.clj [hunk 2]
```diff
@@ -383,6 +383,25 @@
...
+                   :trace {:base 3
+                           :successful {:msg "give the Runner 1 tag"
+                                        :delayed-completion true
```
Comment (nealterrell) [inline src/clj/game/cards/operations.clj]: This key was renamed to `:async`.

File: src/clj/game/cards/operations.clj (hunk 1 above)
Comment (msbeck) [reply src/clj/game/cards/operations.clj]: done

File: src/clj/game/cards/operations.clj (hunk 2 above)
Comment (msbeck) [reply src/clj/game/cards/operations.clj]: removed the first `:delayed-completion` and changed the second to `:async`

**Pull request merged**